azure-mgmt-botservice/tests/test_mgmt_botservice_channels.py

```diff
@@ -0,0 +1,178 @@
+from devtools_testutils import AzureMgmtTestCase, ResourceGroupPreparer
+from azure.mgmt.botservice import AzureBotService
+from azure.mgmt.botservice.models import Bot, BotProperties,sku,BotChannel,ErrorException
+from azure.mgmt.botservice.models import (
+    Bot,
+    BotProperties,
+    BotChannel,
+    ErrorException,
+    sku
+)
+
+class BotServiceChannelsTestCase(AzureMgmtTestCase):
+    def setUp(self):
+        super(BotServiceChannelsTestCase, self).setUp()
+        #create a bot here
+        self.client = self.create_mgmt_client(AzureBotService)
+        self.resource_name = self.get_resource_name('azurebotservice')
+    
+    def createBot(self):
+        location = 'global'
+        sku_name = 'Free'
+        kind= 'Bot'
+        display_name = "this is a test bot"
+        description= "this is a description for a test bot"
+        endpoint = "https://bing.com/messages/"
+        msa_app_id = "056d9ad9-17a9-4cc7-aebb-43bf6f293a08"
+        developer_app_insight_key = '59513bad-10a7-4d41-b4d0-b1c34c6af511'
+        developer_app_insights_api_key = 'w24iw5ocbhcig71su7ibaj63hey5ieaozeuwdv11'
+        developer_app_insights_application_id = 'cf03484e-3fdb-4b5e-9ad7-94bde32e5a11'
+        bot = self.client.bots.create(
+            resource_group_name = self.resource_group_name,
+            resource_name = self.resource_name,
+            parameters = Bot(
+                location= location,
+                sku = sku.Sku(sku_name),
+                kind= kind,
+                properties= BotProperties(
+                    display_name = display_name,
+                    description= description,
+                    endpoint = endpoint,
+                    msa_app_id = msa_app_id,
+                    developer_app_insight_key = developer_app_insight_key,
+                    developer_app_insights_api_key = developer_app_insights_api_key,
+                    developer_app_insights_application_id = developer_app_insights_application_id,
+                )
+            )
+        )
+    
+    def tearDown(self):
+        super(BotServiceChannelsTestCase, self).tearDown()
+        self.client.bots.delete(
+            resource_group_name = self.resource_group_name,
```

Comment:
you don't need that, that's the role of the preparer

Reply:
i dont have a bot preparer right now. i will add one in my next review when i enable the skype, teams tests.